variables.tf

```diff
@@ -439,6 +451,44 @@ variable "k8s_metadata_cleanup_ingress_path" {
   type        = string
 }
 
+# K8S - API deployment
+
+variable "k8s_api_memory" {
+  description = "Total memory to allot to the Beekeeper API pod."
+  default     = "2Gi"
```

Comment:
this is based on?

Reply:
I expect this to take no memory so should be ok to run on minimum pod?

Reply:
I'm going to leave it as it is because in kubernetes it uses the same memory as the other pods, so it makes sense to have the same total memory than the others

Reply:
For comparison, each Cloverleaf Job uses 1 Gi and taskrunner uses 2 Gi. So, I think you should be fine with 1 Gi here.

Reply:
@shermosa I think you marked this as resolved but you didn't change it